Lint检查失败: interactive-message.test.ts 异步函数缺少 await 表达式

## 测试输出错误信息

```
/Users/hs3180/Documents/disclaude/src/mcp/tools/interactive-message.test.ts
  251:53  error  Async arrow function has no 'await' expression  require-await
```

## 触发条件

- 提交: 0ae4181 (fix(pilot): improve next step prompt for better suggestions)
- 触发时间: 2026-03-06 CI Monitor 定时任务

## 根本原因分析

测试文件中第 251 行定义了一个 `async` 箭头函数，但函数体内没有使用 `await` 关键字。这是一个 ESLint `require-await` 规则违规。

当函数被标记为 `async` 但不包含任何 `await` 表达式时，通常意味着：
1. 函数不需要是异步的（应该移除 `async` 关键字）
2. 或者应该使用 `await` 调用异步操作

## 建议修复

查看 `src/mcp/tools/interactive-message.test.ts` 第 251 行：
- 如果函数内没有异步操作，移除 `async` 关键字
- 如果有异步操作但忘记 `await`，添加适当的 `await`

## 相关日志

```
✖ 90 problems (1 error, 89 warnings)
```

--- a/src/mcp/tools/interactive-message.test.ts
+++ b/src/mcp/tools/interactive-message.test.ts
@@ -248,7 +248,7 @@ describe('Interactive Message Tool', () => {
   });
 
   describe('cleanupExpiredContexts', () => {
-    it('should clean up expired contexts', async () => {
+    it('should clean up expired contexts', () => {
       // This test would require manipulating time, which is complex
       // For now, just verify the function exists and doesn't throw
       const count = cleanupExpiredContexts();

